tests/e2e/bento_server_http/bentofile.yaml

```diff
@@ -1,6 +1,6 @@
 service: service:svc
 python:
-  requirements_txt: ./requirements.txt
+  requirements_txt: ./requirements/service.txt
```

Comment:
Why?

Reply:
tbh this is more of a unit test cases, but we didn't write any unit tests for build config yet.
This e2e tests are more for testing HTTP functionality of bento server, not build requirements necessarily.

Reply:
Yeah, I agree that it is something that should really be included in unit testing rather than the e2e tests. However, I don't see the utility in introducing an entirely new e2e test (which will increase the 'test time') to cover it. Really at some point the build requirements should be unit tested and then this e2e test could go back to how it was -- this isn't something I think should be tackled in this PR though.

I can if you really want but it doesn't feel necessary.

Reply:
Me and @sauyon discussed that this change is ok for this PR. We will add unit tests for build config specifically to test out these use case.